src/Check/ApcFragmentation.php

```diff
@@ -90,11 +90,12 @@ public function check()
             return new Skip('APC has not been enabled in CLI.');
         }
 
-        if (! function_exists('apc_sma_info')) {
+        $memoryAllocationMethod = \PHP_VERSION_ID < 70000 ? 'apc_sma_info' : 'apcu_sma_info';
```

Comment:
In attempting to test this on PHP 5.6, I discovered that APC _cannot compile against PHP 5.5 and later_! Since this project only supports 5.6 and up, we can check here for only existence of the APCu extension. I'll make that change during merge.